### DISCUSSION TITLE
hands-on-sessions

### DISCUSSION THREAD

**Discussion by nuveyre** [CSIprinceton/workshop-july-2022#5] [Q&A] [answered]
Dear committee,

Do these workshop tutorials details stay in githup? If it is, I will try at any time. Because downloading has a long process for my computer. 

Thank you

Best Regards
Nuveyre Canbolat

**Comment by PabloPiaggi** [comment, marked as answer]
Yes, you will still find the tutorials here after the workshop.

**Reply by nuveyre** [reply]
Thank you

**Comment by Yi-FanLi** [comment]
Hi Nuveyre,
You are welcome to use the resources after the workshop. Also, if you have any questions when learning DP, you can use the Discussion part of this repo to ask them. You can "watch" this repo to find any further updates.

Thanks,
Yifan

**Comment by nuveyre** [comment]
Hi,
I need to leave early today. I may miss some of the tutorials. Therefore, I
want to thank you and the whole committee members for this amazing
workshop. In future, If I need to ask a question, I can use the Discussion
part of this repo.

Thank you again.

Best regards

Nüveyre Canbolat


Yifan Li李一帆 ***@***.***>, 8 Tem 2022 Cum, 17:31 tarihinde
şunu yazdı:

> Hi Nuveyre,
> You are welcome to use the resources after the workshop. Also, if you have
> any questions when learning DP, you can use the Discussion part of this
> repo to ask them. You can "watch" this repo to find any further updates.
>
> Thanks,
> Yifan
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CSIprinceton/workshop-july-2022/discussions/5#discussioncomment-3108370>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWTGZR2UJWS7QZVQVNAH7OLVTA3VXANCNFSM526BQ6HQ>
> .
> You are receiving this because you authored the thread.Message ID:
> <CSIprinceton/workshop-july-2022/repo-discussions/5/comments/3108370@
> github.com>
>


-- 
------------------------------------------------------
Nüveyre Canbolat
Gazi University
Dept. of Chemistry
Ankara, Turkey
